initial/pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.springframework</groupId>
-    <artifactId>draft-gs-template</artifactId>
+    <artifactId>spring-cloud-gcp-samples</artifactId>
```

Comment:
the maven artifactId should be the same as the baseName in gradle and the project-id (spring-cloud-gcp). There is a reference to the jar file in the "Making executable section that doesn't work for the maven jar file name as it is because of this mismatch.